crates/providers/lattice-controller/.gitignore

```diff
@@ -1,26 +1,16 @@
-# This file lists build byproducts, 
+# This file lists build byproducts,
 # IDE-specific files (unless shared by your team)
 
-#
-
 ## Build
-/build
-/dist/
 /target
 **target
 
-## File system
-.DS_Store
-desktop.ini
-
 ## Editor
 *.swp
 *.swo
 Session.vim
 .cproject
-.idea
 *.iml
-.vscode
```

Comment:
Same comment about keeping these files around

Reply:
Will do!

- [ ] re-add `.idea` to `lattice-controller/.gitignore`